client/src/app/editor/file-outputs/file-outputs.component.ts

```diff
@@ -28,11 +26,16 @@ export class OutputFilesDataSource extends DataSource<any> {
 
   disconnect() {}
 }
-
 @Component({
   selector: 'ml-file-outputs',
   templateUrl: './file-outputs.component.html',
-  styleUrls: ['./file-outputs.component.scss']
+  styleUrls: ['./file-outputs.component.scss'],
+  animations: [
```

Comment:
Introducing the state here (`showing`) is not really necessary, because you only have this one state. The `group` is also not necessary because you only run a single animation. As mentioned earlier, I would apply an animation to the `mat-table` element and then query all `mat-row` elements to nicely stagger them. This is what I would most likely do:

```
trigger('staggerIn', [
transition('* <=> *', [
  query('mat-row', style({ opacity: 0 }), { optional: true }),
  query(
    'mat-row',
    stagger(100, [
      style({ opacity: 0 }),
      animate('300ms cubic-bezier(0.35, 0, 0.25, 1)', style({ opacity: 1 }))
    ]),
    { optional: true }
  )
])
```

Note that this ^ is a very specific animation. You could also make it more reusable but for now I think this is fine. If you find yourself adding this animation to other elements in our app then you can come back and think about ways to make it more reusable.